src/Features/Core/Portable/FindUsages/AbstractFindUsagesService.ProgressAdapter.cs

```diff
@@ -107,20 +108,26 @@ public async ValueTask OnDefinitionFoundAsync(SymbolGroup group, CancellationTok
         }
 
         public async ValueTask OnReferencesFoundAsync(
-            ImmutableArray<(SymbolGroup group, ISymbol symbol, ReferenceLocation location)> references, CancellationToken cancellationToken)
+            IAsyncEnumerable<(SymbolGroup group, ISymbol symbol, ReferenceLocation location)> references, CancellationToken cancellationToken)
```

Comment:
another API is also moving to streaming, instead of chunking.  Since it itself will have this ProducerCOnsumer.RunParallel in it.  So it can just consume whatever is thrown at it, chunking it itself.